src/yuzu/discord_impl.cpp

```diff
@@ -34,16 +35,31 @@ void DiscordImpl::Update() {
                          std::chrono::system_clock::now().time_since_epoch())
                          .count();
     std::string title;
+    u64 titleId;
+
     if (system.IsPoweredOn()) {
         system.GetAppLoader().ReadTitle(title);
+        system.GetAppLoader().ReadProgramId(titleId);
     }
+
     DiscordRichPresence presence{};
-    presence.largeImageKey = "yuzu_logo";
-    presence.largeImageText = "yuzu is an emulator for the Nintendo Switch";
+
     if (system.IsPoweredOn()) {
+        std::string gameCoverUrl = "https://tinfoil.media/ti/";
+        gameCoverUrl += QString::fromStdString(fmt::format("{:016X}", titleId)).toStdString();
```

Comment:
This can be simplified as:

```suggestion
        gameCoverUrl += fmt::format("{:016X}", titleId);
```